README.md

```diff
@@ -34,70 +34,63 @@
 </details>
 
 ### Key Features <a name="key-features"></a>
-
-- OOP school library: decorate a class
 - OOP school library: set up associations
 - OOP school library: add basic UI
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
-## 🚀 You can check the Live Demo here<a name="live-demo"></a>
-
-The live demo will be availabe soon
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- GETTING STARTED -->
 
-## 💻 Getting Started <a name="getting-started"></a>
-
-- Step 1: Clone this project using this git comand "git clone", (if you don't have git installed please visit
-  https://git-scm.com/downloads and install it).
-- Step 2: Once you have installed git you can clone this project using the command below:
-  ```
-  git clone https://github.com/Gardimy/OOP-school-library
-  ```
-- Step 3: Once you have cloned this repo navigate to the folder that contains
-  the repository.
-- Step 4: If you want to watch the html file open it with your favorite web browser(This project MUST be opened from VS Code for full functionality).
-- Step 5: If you want to edit this project open it with your favorite text editor.
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-## 💻 Getting Started <a name="getting-started"></a>
+## Getting Started
 
-To get a local copy up and running cloning this repo `https://github.com/Gardimy/OOP-school-library`.
+To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
-There are no known prerequisites to run this project.
-If you encounter any issues trying to run it, please feel free to
-[open a ticket](https://github.com/Gardimy/OOP-school-library/issues) to let me know and i will work on it to the best
-of my abilities
+> You need the following tools be installed in your computer:
+
+> - [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/)
+> - [Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/articles/ruby_installation_instructions.md)
+> - IDE
+> - Etc
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-`git clone https://github.com/Gardimy/OOP-school-library`
+```sh
+  cd my-prefered-folder
+  
+  git clone https://github.com/Gardimy/OOP-school-library.git
 
-or
+```
 
-`git clone https://github.com/Gardimy/OOP-school-library` <-- _Only if you have [SSH keys configured](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) on your GitHub account_.
+### Install
 
-### Usage
+Install this project with:
 
-To run the project:
+```sh
+  cd OOP-school-library
+  
+  bundle install
```

Comment:
- [ ] [OPTIONAL]
Let's take note that the command to install ruby dependencies is `bundle install` or `gem install gem-name`.